hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/HederaConfig.java

```diff
@@ -25,24 +26,32 @@ public record HederaConfig(
         @ConfigProperty(defaultValue = "1001") @NetworkProperty long firstUserEntity,
         @ConfigProperty(defaultValue = "0") @NetworkProperty long realm,
         @ConfigProperty(defaultValue = "0") @NetworkProperty long shard,
```

Comment:
These two might change.  It's not entirely clear to me if a *node* belongs to a shard and realm or if the entire *network* implements a single shard and realm.  I rather suspect the former is more likely, with the nuance that a single node might join multiple shards and/or realms, and a network comprises all of the shards and realms extant on the ledger.

Reply:
True! The shard and realm will be different for different nodes on the same network. A node will only be part of a single shard at a time, but could move from shard to shard each day. For now, having these in config is OK I think, but in the future when we do sharding, we will remove them and supply them through some other means.

Reply:
Changed them to node properties.